ClinGen/clincoded
# Connects to #1631. New users and new affiliations.

**Steps to test:**
1. Sign in as admin and associate yourself with the affiliations 10021, 10022, 10023 and 10024.
2. Sign out and sign in again. Expect to see the affiliation modal to be shown for you to choose any one of the affiliations described above.
3. Go to **^/users** page and filter for the last name **Popejoy**. Expect to see one entry to be returned in the search result. You may repeat this test step with any of the new users in R17.
4. Go to ^/add-curator page and expect to see a new field for adding affiliation(s).
5. Enter "10007, 10016" in the affiliation text field and other required information. Expect to be able to successfully submit the form.
